extension/src/uddsketch.rs

```diff
@@ -335,4 +370,57 @@ mod tests {
             }
         });
     }
+
+    #[pg_test]
+    fn test_compound_agg() {
+        Spi::execute(|client| {
+            client.select("CREATE TABLE new_test (device INTEGER, value DOUBLE PRECISION)", None, None);
+            client.select("INSERT INTO new_test SELECT dev, dev - v FROM generate_series(1,10) dev, generate_series(0, 1.0, 0.01) v", None, None);
+
+            let sanity = client
+                .select("SELECT COUNT(*) FROM new_test", None, None)
+                .first()
+                .get_one::<i32>();
+            assert_eq!(Some(1010), sanity);
+
+            client.select(
+                "SET timescale_analytics_acknowledge_auto_drop TO 'true'",
+                None,
+                None,
+            );
+
+            client.select("CREATE VIEW sketches AS \
+                SELECT device, timescale_analytics_experimental.uddsketch(20, 0.01, value) \
+                FROM new_test \
+                GROUP BY device", None, None);
+
+            client.select("CREATE VIEW composite AS \
+                SELECT timescale_analytics_experimental.uddsketch(uddsketch) \
+                FROM sketches", None, None);
+
+            client.select("CREATE VIEW base AS \
+                SELECT timescale_analytics_experimental.uddsketch(20, 0.01, value) \
+                FROM new_test", None, None);
+                
+            let (value, error) = client
+                .select("SELECT \
+                    timescale_analytics_experimental.quantile(uddsketch, 0.9), \
+                    timescale_analytics_experimental.error(uddsketch) \
+                    FROM base", None, None)
+                .first()
+                .get_two::<f64, f64>();
+                
+            let (test_value, test_error) = client
+                .select("SELECT \
+                    timescale_analytics_experimental.quantile(uddsketch, 0.9), \
+                    timescale_analytics_experimental.error(uddsketch) \
+                    FROM composite", None, None)
+                .first()
+                .get_two::<f64, f64>();
+
+            apx_eql(test_value.unwrap(), value.unwrap(), 0.0001);
```

Comment:
I think for uddsketch, the merges shouldn't matter and the result is deterministic whether or not we have the intermediate result, so the apx_eql shouldn't be necessary?